api/app/Models/PoolCandidateFilter.php

```diff
@@ -57,6 +57,10 @@ public function poolCandidateSearchRequest(): HasOne
     {
         return $this->hasOne(PoolCandidateSearchRequest::class);
     }
+    public function stream()
+    {
+        return $this->stream;
+    }
```

Comment:
what is the purpose of this given pool_candidate_filters is obsolete and not being updated here?

Reply:
Maybe if this is the expectation, we could use phpdoc to deprecate the class and explain that we're not expecting any more changes to them?

https://phpstan.org/writing-php-code/phpdocs-basics#deprecations